src/blockchain_db/sqlite/db_sqlite.cpp

```diff
@@ -172,6 +172,47 @@ namespace cryptonote {
 
       transaction.commit();
     }
+
+    bool have_archive = false;
+    SQLite::Statement archive_cols{db, "PRAGMA main.table_info(batched_payments_accrued_archive)"};
```

Comment:
per discussion with @jagerman: `SELECT COUNT(*) FROM sqlite_master WHERE type='table' AND name='batched_payments_accrued_archive';` might be a cleaner way to do this?

Reply:
see also line 126 above